packages/shopping/src/application.ts

```diff
@@ -47,6 +48,13 @@ export class ShoppingApplication extends BootMixin(
 ) {
   constructor(options?: ApplicationConfig) {
     super(options);
+    this.api({
+      openapi: '3.0.0',
+      info: {title: pkg.name, version: pkg.version},
+      paths: {},
```

Comment:
servers: [{url: '/'}]

Reply:
or config it when create the rest app:

https://loopback.io/doc/en/lb4/Server.html#customize-how-openapi-spec-is-served